### ISSUE TITLE
Texture missing

### ISSUE THREAD

**Issue by Tyrantin** [simeonradivoev/MatterOverdrive#667]
Not sure if it was intentional or not, however I figured I would report what I see, or in this case not see!
![2017-03-25](https://cloud.githubusercontent.com/assets/8041616/24320151/fbedcfec-1105-11e7-895d-db2e9d73cfcc.png)

**Comment by shadowfacts** [comment]
Yes, the CTM is broken in 1.10.2 so the Tritanium Glass is rendering without a model.

**Comment by ETStareak** [comment]
And all the guns don't have modle and  texture，too.Also Gravitational Stabilizer，Color Module (Gray)and Contract Market.
But the new modle of Starmap and other things are so great!

**Comment by shadowfacts** [comment]
The missing weapon models has already been fixed.

**Comment by ETStareak** [comment]
Not in rel versions==
And when will 0.5.2 come out?The Chinese lang which joccob uploaded hasn't finished. He said he won't keep translating and allowed me to keep on work.So I want to know when shuold I upload the lang.

**Comment by shadowfacts** [comment]
I don't know, but the localization likely won't change, so feel free to work on it.

**Comment by Tyrantin** [comment]
Question, Will there be an update within the next 3-4 days? Would like to move the mod pack to Curse.

**Comment by Muirrum** [comment]
Most likely not. I am working on this.
